app/assets/stylesheets/rentals/rental_schedule.css

```diff
@@ -17,7 +17,8 @@ table, th, td {
 }
 
 .simple-calendar table td {
-  height: 100px !important;
+  height: 123px !important;
+  width : 163px !important;
```

Comment:
Does this work without the `!important`? I am curious because this should be pretty high up the priority stack (for CSS).